packages/client/src/utils/countryUtils.ts

```diff
@@ -0,0 +1,334 @@
+  ['Saint Kitts and Nevis', 'kn', '1', 18, ['869']],
+  ['Saint Lucia', 'lc', '1', 19, ['758']],
+  ['Saint Martin (Saint-Martin (partie française))', 'mf', '590', 2],
+  ['Saint Pierre and Miquelon (Saint-Pierre-et-Miquelon)', 'pm', '508'],
+  ['Saint Vincent and the Grenadines', 'vc', '1', 20, ['784']],
+  ['Samoa', 'ws', '685'],
+  ['San Marino', 'sm', '378'],
+  ['São Tomé and Príncipe (São Tomé e Príncipe)', 'st', '239'],
+  ['Saudi Arabia (‫المملكة العربية السعودية‬‎)', 'sa', '966'],
+  ['Senegal (Sénégal)', 'sn', '221'],
+  ['Serbia (Србија)', 'rs', '381'],
+  ['Seychelles', 'sc', '248'],
+  ['Sierra Leone', 'sl', '232'],
+  ['Singapore', 'sg', '65'],
+  ['Sint Maarten', 'sx', '1', 21, ['721']],
+  ['Slovakia (Slovensko)', 'sk', '421'],
+  ['Slovenia (Slovenija)', 'si', '386'],
+  ['Solomon Islands', 'sb', '677'],
+  ['Somalia (Soomaaliya)', 'so', '252'],
+  ['South Africa', 'za', '27'],
+  ['South Korea (대한민국)', 'kr', '82'],
+  ['South Sudan (‫جنوب السودان‬‎)', 'ss', '211'],
+  ['Spain (España)', 'es', '34'],
+  ['Sri Lanka (ශ්‍රී ලංකාව)', 'lk', '94'],
+  ['Sudan (‫السودان‬‎)', 'sd', '249'],
+  ['Suriname', 'sr', '597'],
+  ['Svalbard and Jan Mayen', 'sj', '47', 1, ['79']],
+  ['Sweden (Sverige)', 'se', '46'],
+  ['Switzerland (Schweiz)', 'ch', '41'],
+  ['Syria (‫سوريا‬‎)', 'sy', '963'],
+  ['Taiwan (台灣)', 'tw', '886'],
+  ['Tajikistan', 'tj', '992'],
+  ['Tanzania', 'tz', '255'],
+  ['Thailand (ไทย)', 'th', '66'],
+  ['Timor-Leste', 'tl', '670'],
+  ['Togo', 'tg', '228'],
+  ['Tokelau', 'tk', '690'],
+  ['Tonga', 'to', '676'],
+  ['Trinidad and Tobago', 'tt', '1', 22, ['868']],
+  ['Tunisia (‫تونس‬‎)', 'tn', '216'],
+  ['Turkey (Türkiye)', 'tr', '90'],
+  ['Turkmenistan', 'tm', '993'],
+  ['Turks and Caicos Islands', 'tc', '1', 23, ['649']],
+  ['Tuvalu', 'tv', '688'],
+  ['U.S. Virgin Islands', 'vi', '1', 24, ['340']],
+  ['Uganda', 'ug', '256'],
+  ['Ukraine (Україна)', 'ua', '380'],
+  ['United Arab Emirates (‫الإمارات العربية المتحدة‬‎)', 'ae', '971'],
+  ['United Kingdom', 'gb', '44', 0],
+  ['United States', 'us', '1', 0],
+  ['Uruguay', 'uy', '598'],
+  ['Uzbekistan (Oʻzbekiston)', 'uz', '998'],
+  ['Vanuatu', 'vu', '678'],
+  ['Vatican City (Città del Vaticano)', 'va', '39', 1, ['06698']],
+  ['Venezuela', 've', '58'],
+  ['Vietnam (Việt Nam)', 'vn', '84'],
+  ['Wallis and Futuna (Wallis-et-Futuna)', 'wf', '681'],
+  ['Western Sahara (‫الصحراء الغربية‬‎)', 'eh', '212', 1, ['5288', '5289']],
+  ['Yemen (‫اليمن‬‎)', 'ye', '967'],
+  ['Zambia', 'zm', '260'],
+  ['Zimbabwe', 'zw', '263'],
+  ['Åland Islands', 'ax', '358', 1, ['18']]
+]
+
+// loop over all the countries above, restructuring the data to be objects with named keys
+for (let i = 0; i < countries.length; i++) {
+  const c = countries[i]
+  countries[i] = {
+    name: c[0],
+    iso2: c[1],
+    dialCode: c[2],
+    priority: c[3] || 0,
+    areaCodes: c[4] || null
+  }
+}
+
+interface ICountryData {
+  name: string
+  iso2: string
+  dialCode: string
+  priority: number
```

Comment:
Priority seems to be a number in the reference data list above. Let's define it as a string and use a default value of `'0'` on line 321

Reply:
got it